server/src/weaverbird/pipeline/pipeline.py

```diff
@@ -184,6 +194,79 @@ def dict(self, *, exclude_none: bool = True, **kwargs) -> dict:
 ]
 
 
+def _remove_void_from_combo_condition(
+    condition: ConditionComboAnd | ConditionComboOr | None, combo: str
+) -> ConditionComboAnd | ConditionComboOr | None:
+    """
+    Loop into a combo condition and search for embedded condition that may
+    contains __VOID__ as column/value
+    """
+    if condition is None:
+        return None
+
+    loop_on: list[Condition] = []
```

Comment:
We're not looping on this, I don't understand the name.. How about `sanitized_conditions` ?